app/containers/App/index.js

```diff
@@ -27,14 +27,23 @@ function App({ location }) {
     <ThemeProvider theme={theme}>
       <div>
         <Switch>
-          {_.map(Object.keys(routeConfig), (routeKey, index) => (
-            <Route
-              exact={routeConfig[routeKey].exact}
-              key={index}
-              path={location.pathname + routeConfig[routeKey].route}
-              component={routeConfig[routeKey].component}
```

Comment:
also i don't see the changes in the routeConfig in your branch 
https://github.com/wednesday-solutions/react-template/blob/feature/add-props-to-router/app/routeConfig.js#L1

can you please update the routeConfig as well

Reply:
will do sir